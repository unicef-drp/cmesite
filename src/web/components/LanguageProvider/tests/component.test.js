import React from 'react';
import { shallow } from 'enzyme';
import Component from '../component';

const props = {
  messages: {},
  locale: 'en',
};

describe('app |Â App | component', () => {
  describe('render', () => {
    it('should match snapshot', () => {
      const root = (
        <Component {...props}>
          <div />
        </Component>
      );

      const wrapper = shallow(root);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
