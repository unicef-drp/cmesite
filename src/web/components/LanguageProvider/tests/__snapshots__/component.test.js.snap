// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app |Â App | component render should match snapshot 1`] = `
<IntlProvider
  key="en"
  locale="en"
>
  <div />
</IntlProvider>
`;
